src/components/Posts/Post/Post.js

```diff
@@ -11,11 +11,11 @@ import { AiOutlineHeart } from 'react-icons/ai';
 const Post = ({ name, text, createdAt, likes, postId, postedBy, comments }) => {
   const [open, setOpen] = useState(false);
   const user = JSON.parse(localStorage.getItem("user"));
-  const [postLikedByCurrentUser, setPostLikedByCurrentUser] = useState(likes.includes(user._id) ? <FcLike key={1} /> : <AiOutlineHeart key={2} />)
+  const [postLikedByCurrentUser, setPostLikedByCurrentUser] = useState(likes.includes(user?._id) ? <FcLike key={1} /> : <AiOutlineHeart key={2} />)
   const onHandleLike = (event) => {
     setPostLikedByCurrentUser(postLikedByCurrentUser.key === "1" ? <AiOutlineHeart key={2} /> : <FcLike key={1} />);
     const config = {
-      headers: { Authorization: `Bearer ${user.token}` },
+      headers: { Authorization: `Bearer ${user?.token}` },
```

Comment:
Search for axios instance, so you can add headers in one place only and that instance can be used for URL and data only. other things can be done there